cl/recap/admin.py

```diff
@@ -70,4 +74,32 @@ class PacerHtmlFilesAdmin(CursorPaginatorAdmin):
     )
 
 
+@admin.action(description="Reprocess selected recap.emails")
+def reprocess_failed_epq(modeladmin, request, queryset):
+    recap_email_user = User.objects.get(username="recap-email")
+    for epq in queryset:
+        do_recap_document_fetch(epq, recap_email_user)
+
+    modeladmin.message_user(
+        request,
+        ngettext(
+            "%d epq was successfully reprocessed.",
+            "%d epqs were successfully reprocessed.",
+            queryset.count(),
+        )
+        % queryset.count(),
+        messages.SUCCESS,
+    )
+
+
+@admin.register(EmailProcessingQueue)
+class EmailProcessingQueueAdmin(CursorPaginatorAdmin):
+    list_display = (
+        "__str__",
+        "status",
+    )
+    list_filter = ("status",)
+    actions = [reprocess_failed_epq]
+
```

Comment:
```suggestion
raw_id_fields = (
        "recap_documents",
    )
```

We should use a `raw_id` field to display the `RECAPDocuments` to avoid loading all the RDs from DB.